layers/dot11.go

```diff
@@ -1796,6 +1796,10 @@ func (m *Dot11MgmtReassociationResp) CanDecode() gopacket.LayerClass {
 func (m *Dot11MgmtReassociationResp) NextLayerType() gopacket.LayerType {
 	return LayerTypeDot11InformationElement
 }
+func (m *Dot11MgmtReassociationResp) DecodeFromBytes(data []byte, df gopacket.DecodeFeedback) error {
+	m.Payload = data
```

Comment:
This type has fixed parameters

Reply:
Fixed- I simply mirrored over the parsing code for association response frames, since they have identical- and identically formatted- fixed paramaters.